config/initializers/ember.rb

```diff
@@ -1,5 +1,3 @@
 EmberCLI.configure do |c|
-  c.app :exam_builder,
-    bower_path: "bin/bower",
-    exclude_ember_deps: ["jquery", "handlebars"]
+  c.app :exam_builder
```

Comment:
* We couldn't properly reproduce the bug in this repo, since
  `bower_path: bin/bower` doesn't exist, and `jquery` and `handlebars`
  aren't included in your Sprockets configuration. We've removed them
  here for the sake of reproducing, but in your production app they can
  stick around, so long as `bower_path` points to a valid `bower`
  executable, and `jquery` and `handlebars` come from your Sprockets
  configuration